trnsysGUI/mainWindow.py

```diff
@@ -223,6 +234,7 @@ def askForSaveAndLoadSettings() -> None:
         self.mb.addMenu(self.fileMenu)
         self.mb.addMenu(self.projectMenu)
         self.mb.addMenu(self.editMenu)
+        self.mb.addMenu(self.hydraulicModesMenu)
```

Comment:
The mainwindow should be reduced to something like these few lines.

self.mb.addMenu(getHydraulicModesMenu(self)).

The rest can then be moved to a menu module.